release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -89,6 +89,12 @@ For `console` and `downloads` routes, custom routes functionally is now implemen
 [id="ocp-4-8-networking"]
 === Networking
 
+[id="ocp-4-8-network-policy-host-network-egress-router-pods"]
+==== Network policy supports selecting host network egress router pods
+
+When using the OVN-Kubernetes cluster network provider, you can select egress router pods in network policy rules with the `network.openshift.io/policy-group: ingress` namespace selector. Additionally, when using either OVN-Kubernetes or the OpenShift SDN cluster network provider, you can use the `network.openshift.io/policy-group: host-network` to select all host network traffic.
```

Comment:
From the user's point of view, these are unrelated features, so I don't know if it makes sense to document them under the same sub-heading.

For the first feature (`policy-group: ingress`), the behavior change affects both openshift-sdn and ovn-kubernetes; the end result is that they now implement exactly the same thing ("you can select router pods using the ... namespace selector, even in clusters that use routers in HostNetwork mode"). If you just want to focus more on the _change_ in behavior since this is the release notes, then for openshift-sdn, the change is that in 4.7 and earlier, when using routers in HostNetwork mode, you needed to do some extra work (adding labels to the default namespace) in order to make the NetworkPolicy work, and in 4.8 that's no longer necessary. For ovn-kubernetes, the change is that in 4.7 and earlier, when using routers in HostNetwork mode, there was no way to make it work at all, but now it does work.